bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -467,6 +474,38 @@ impl Room {
     pub async fn typing_notice(&self, is_typing: bool) -> Result<(), ClientError> {
         Ok(self.inner.typing_notice(is_typing).await?)
     }
+
+    pub async fn power_level_settings(&self) -> Result<RoomPowerLevelSettings, ClientError> {
+        let power_levels = self.inner.get_room_power_levels().await?;
+        Ok(power_levels.into())
+    }
+
+    pub async fn update_power_level_settings(
+        &self,
+        updates: RoomPowerLevelSettings,
+    ) -> Result<(), ClientError> {
+        let mut power_levels = self.inner.get_room_power_levels().await?;
+        power_levels.apply(updates)?;
+        self.inner
+            .send_state_event(RoomPowerLevelsEventContent::from(power_levels))
```

Comment:
Would it be better for this to call a new `inner.update_power_levels` method rather than sending the state directly?